jsonnet/openshift-state-metrics.libsonnet

```diff
@@ -18,6 +18,13 @@ function(params) {
   clusterRoleBinding: osm.openshiftStateMetrics.clusterRoleBinding,
   clusterRole: osm.openshiftStateMetrics.clusterRole,
   deployment: osm.openshiftStateMetrics.deployment {
+    metadata+: {
+      labels+: {
+        'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
+        'app.kubernetes.io/name': 'openshift-state-metrics',
+        'app.kubernetes.io/component': 'exporter',
```

Comment:
These two should be moved to https://github.com/openshift/openshift-state-metrics/pull/75/ and we should add 'app.kubernetes.io/part-of': 'openshift-monitoring' here. The goal is to have everything generic "upstream" and everything CMO specific here.